app/assets/scripts/game/render.js

```diff
@@ -210,3 +221,62 @@ const setStealablePickomino = (data) => {
     stealablePickominoElement.onclick = () => stealPlayerPickomino(stealablePlayerRanking, data)
   }
 }
+
+const setSaveButton = (value, data, id) => {
+  const saveButton = document.getElementById(id)
+  if (value) {
+    saveButton.onclick = () => {
+      axios({
+        method: 'post',
+        url: '/save',
+        data: JSON.stringify(data)
+      })
+      setSaveButton(false, data)
+    }
+    saveButton.removeAttribute('disabled')
+  } else {
+    saveButton.onclick = null
+    saveButton.setAttribute('disabled', '')
+  }
+}
+// Modal creation
+const modalElement = (value) => htmlToElement(`
+<div class="modal" id="showmodal">
+  <div class="modal-background"></div>
+  <div class="modal-content">
+      <div class="section modal-wrap" >
+          <div class="column has-text-centered">
+              <p>Voulez vous sauvegarder la partie avant de quitter le jeu?</p>
+              <a href="/save" class="button is-success mt-2" id="save" >OUI, je sauvegarde</a>
+              <a href="${value}" class="button is-danger mt-2">Non, merci</a>
+          </div>
+      </div>
+  </div>
+  <button class="modal-close is-large" aria-label="close" id="cancel"></button>
+</div>`
+)
+// function to bring up the modal
+const modalSaveGame = (valeur, data, id) => {
+  // all internal links of the app
+  const anchorTemplate = document.getElementsByClassName('anchorTemplate')
+  // empty div
+  const savemodal = document.getElementById('saveModal')
+
+  // for each element having the class anchorTemplate
+  for (let i = 0, len = anchorTemplate.length; i < len; i++) {
+    // open modal
+    anchorTemplate[i].onclick = () => {
+      savemodal.appendChild(modalElement(anchorTemplate[i].value))
+      const modal = document.getElementById('showmodal')
+      modal.style.display = 'block'
```

Comment:
Bulma permet de rendre une modale visible en utilisant la class `is-active`

Reply:
Oui autre méthode possible